changelog-template.hbs

```diff
@@ -0,0 +1,17 @@
+
+{{#each releases}}
+
+  ## [{{title}}] {{date}}
```

Comment:
```suggestion
  ## [{{title}}] - {{date}}
```

Small adjustment for readability and consistency with the keepachangelog.com example